source/acquaman/OLDSGM/SGMXASScanController.cpp

```diff
@@ -0,0 +1,294 @@
+//				qdebug() << "Adding HV turn on to initialization actions";
+				initializationActions_->appendAction(0, config_->allDetectors()->detectorAt(x)->turnOnAction());
+			}
+		}
+		else
+			enableSync = false;
+		int syncIndex = SGMBeamline::sgm()->synchronizedDwellTimeDetectorIndex(config_->allDetectors()->detectorAt(x));
+//		CLSSynchronizedDwellTime *clsDwellTime = qobject_cast<CLSSynchronizedDwellTime*>(SGMBeamline::sgm()->synchronizedDwellTime());
+		if( (syncIndex > 1) && (SGMBeamline::sgm()->synchronizedDwellTime()->enabledAt(syncIndex) != enableSync) ){
+//			tmpBAction = clsDwellTime->elementAt(syncIndex)->createEnableAction(enableSync);
+			//tmpBAction = SGMBeamline::sgm()->synchronizedDwellTime()->elementAt(syncIndex)->createEnableAction(enableSync);
+			tmpBAction ? initializationActions_->appendAction(0, tmpBAction) : cleanupFailed = true;
+		}
+	}
+
+	/*
+	if(config_->allDetectorConfigurations().isActiveNamed(SGMBeamline::sgm()->amptekSDD1()->detectorName()))
+		tmpBAction = SGMBeamline::sgm()->createSDD1EnableAction(true);
+	else
+		tmpBAction = SGMBeamline::sgm()->createSDD1EnableAction(false);
+	tmpBAction ? initializationActions_->appendAction(0, tmpBAction) : initializationFailed = true;
+	if(config_->allDetectorConfigurations().isActiveNamed(SGMBeamline::sgm()->amptekSDD2()->detectorName()))
+		tmpBAction = SGMBeamline::sgm()->createSDD2EnableAction(true);
+	else
+		tmpBAction = SGMBeamline::sgm()->createSDD2EnableAction(false);
+	tmpBAction ? initializationActions_->appendAction(0, tmpBAction) : initializationFailed = true;
+	*/
+
+//	tmpBAction = SGMBeamline::sgm()->synchronizedDwellTime()->createMasterTimeAction(config_->regionTime(0));
+	tmpBAction ? initializationActions_->appendAction(0, tmpBAction) : initializationFailed = true;
+	tmpAction = new AMBeamlineControlMoveAction(SGMBeamline::sgm()->nextDwellTimeTrigger());
+	tmpAction->setSetpoint(0);
+	tmpAction ? initializationActions_->appendAction(0, tmpAction) : initializationFailed = true;
+	tmpAction = new AMBeamlineControlMoveAction(SGMBeamline::sgm()->nextDwellTimeConfirmed());
+	tmpAction->setSetpoint(1);
+	tmpAction ? initializationActions_->appendAction(0, tmpAction) : initializationFailed = true;
+
+	tmpBAction = SGMBeamline::sgm()->scaler()->createStartAction(0);
+	tmpBAction ? initializationActions_->appendAction(0, tmpBAction) : initializationFailed = true;
+	tmpBAction = SGMBeamline::sgm()->scaler()->createContinuousEnableAction(false);
+	tmpBAction ? initializationActions_->appendAction(0, tmpBAction) : initializationFailed = true;
+
+	initializationActions_->appendStage(new QList<AMBeamlineActionItem*>());
+	tmpBAction = SGMBeamline::sgm()->scaler()->createScansPerBufferAction(1);
+	tmpBAction ? initializationActions_->appendAction(1, tmpBAction) : initializationFailed = true;
+	tmpBAction = SGMBeamline::sgm()->scaler()->createTotalScansAction(1);
+	tmpBAction ? initializationActions_->appendAction(1, tmpBAction) : initializationFailed = true;
```

Comment:
These lines are in the OLD SGM code.  This will all likely be deleted after the merge.